src/utils/sound.ts

```diff
@@ -20,7 +20,7 @@ export const isWAV = (filepath: string): boolean => {
 	return path.extname(filepath).toLowerCase() === ".wav"
 }
 
-let isSoundEnabled = true
+let isSoundEnabled = false
```

Comment:
Couldn't help sneaking this one in, just to be safe.

Reply:
feedback from https://discord.com/channels/1275535550845292637/1275535550845292640/1315443495711936634

Alex. wrote:
> Good idea because its overly unecessary for your average user, if they see a task has been complete, they'll already see it via text